src/borg/helpers/parseformat.py

```diff
@@ -540,9 +540,30 @@ def validator(text):
 
 def archivename_validator():
     def validator(text):
+        assert isinstance(text, str)
+        # we make sure that the archive name can be used as directory name (for borg mount)
         text = replace_placeholders(text)
-        if "/" in text or "::" in text or not text:
-            raise argparse.ArgumentTypeError('Invalid archive name: "%s"' % text)
+        if not (0 < len(text) <= 250):  # max path length on windows is rather short
```

Comment:
On Windows 10, the max path limit can be removed by adding a registry key - in fact the Python on Windows installer does this.
https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry
Something like this may help for checking whether this is enabled: https://stackoverflow.com/a/46546731

Reply:
Ah, that's interesting, thanks!

OTOH, it we use the pyinstaller made binary, most windows installations would still have the low limit, right?

Reply:
see comment in updated PR.